src/main/java/bio/terra/workspace/service/migrate/MigrateService.java

```diff
@@ -11,72 +14,72 @@
-     * This is useful for alpha, staging, and production environments where we make some
-     * guarantee that the database schema is in a valid state for applying changesets to.
-     *
-     * @param changesetFile - relative path to the changeset file in the project
-     * @param dataSource - database to operate on
-     */
-    public void upgrade(String changesetFile, DataSource dataSource) {
-        migrateWorker(changesetFile, dataSource, false);
-    }
+  /**
+   * Upgrade applies changesets to move the current state of the database to a new state. This is
+   * useful for alpha, staging, and production environments where we make some guarantee that the
+   * database schema is in a valid state for applying changesets to.
+   *
+   * @param changesetFile - relative path to the changeset file in the project
+   * @param dataSource - database to operate on
+   */
+  public void upgrade(String changesetFile, DataSource dataSource) {
+    migrateWorker(changesetFile, dataSource, false);
+  }
 
-    private void migrateWorker(String changesetFile, DataSource dataSource, boolean initialize) {
-        try (Connection connection = dataSource.getConnection()) {
-            Liquibase liquibase = new Liquibase(changesetFile,
-                    new ClassLoaderResourceAccessor(),
-                    new JdbcConnection(connection));
-            DatabaseChangeLogLock[] locks = liquibase.listLocks();
-            for (DatabaseChangeLogLock lock : locks) {
-                logger.info(String.format("DatabaseChangeLogLock changeSet: %s, id: %s, lockedBy: %s, granted: %s",
-                        changesetFile, lock.getId(), lock.getLockedBy(), lock.getLockGranted()));
+  private void migrateWorker(String changesetFile, DataSource dataSource, boolean initialize) {
+    try (Connection connection = dataSource.getConnection()) {
+      Liquibase liquibase =
+          new Liquibase(
+              changesetFile, new ClassLoaderResourceAccessor(), new JdbcConnection(connection));
+      DatabaseChangeLogLock[] locks = liquibase.listLocks();
+      for (DatabaseChangeLogLock lock : locks) {
+        logger.info(
+            String.format(
+                "DatabaseChangeLogLock changeSet: %s, id: %s, lockedBy: %s, granted: %s",
+                changesetFile, lock.getId(), lock.getLockedBy(), lock.getLockGranted()));
 
-                 // We can get into this state where one of the APIs is running migrations and gets shut down so that
-                 // another API container can run. It will result in a lock that doesn't get released. This is similar
-                 // to the problems we will have from deploying multiple containers at once that try to run migrations.
-                logger.warn("Forcing lock release");
-                liquibase.forceReleaseLocks();
-            }
+        // We can get into this state where one of the APIs is running migrations and gets shut down
+        // so that
```

Comment:
some of these comment format changes are odd, just fyi

Reply:
Good catch - looks like the line wrapping adds newlines but doesn't like removing them. I'll try to manually fix these rq